.circleci/config.yml

```diff
@@ -12,92 +19,284 @@ jobs:
           command: |
-            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
-            EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
-            ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
-            [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && php composer-setup.php --install-dir=/bin --filename=composer || exit 1
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
             composer config -g github-protocols https && composer config -g repo.packagist composer https://packagist.org
-      - when:
-          condition: << parameters.clone-wordpress-develop >>
-          steps:
-            - run: git clone git://develop.git.wordpress.org/ wordpress-develop
-            - run: cp wordpress-develop/wp-tests-config-sample.php wordpress-develop/wp-tests-config.php
-            - run: sed -i 's/localhost/127.0.0.1/g' wordpress-develop/wp-tests-config.php
-      - run: mkdir -p wordpress-develop/src/wp-content/plugins/atomic-blocks
-      - checkout:
-          path: wordpress-develop/src/wp-content/plugins/atomic-blocks
+            composer install --prefer-dist --no-suggest --optimize-autoloader --no-dev
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - atomic-blocks
+
+  npm_build:
+    executor: node-only
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - run:
+          name: Build assets
+          command: |
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+            npm ci
+            npm run build-assets
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - atomic-blocks
+
+  npm_lint:
+    executor: node-only
+    steps:
+      - attach_workspace:
+          at: ~/project
       - run:
-          working_directory: wordpress-develop/src/wp-content/plugins/atomic-blocks/
-          command: composer install --prefer-dist --no-suggest --optimize-autoloader && npm install
+          name: Lint CSS/JS assets
+          command: |
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+            cd ~/project/atomic-blocks
+            ls -la
+            npm ci
+            npm run lint:css:src
+            npm run lint:js
+
+  bundle:
+    executor:
+      name: php
+      php-version: '7.4'
+    steps:
+      - attach_workspace:
+          at: ~/project
+      - mkdir_artifacts
+      - run:
+          name: "Bundle plugin files into a zip"
+          command: |
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+            apk add --no-cache zip
+            wp i18n make-pot . languages/atomic-blocks.pot --exclude=config,node_modules,scripts,vendor,dist/blocks.build.js --path=/root/project/wordpress-develop/src --allow-root
```

Comment:
Just realized this needs to be moved to another job so it's available to svn. I originally set the svn deploy jobs to require bundle just for the sake of completeness of the build pipeline, not because it really needs bundle.